### PR TITLE
Syncing from upstream OCA/server-backend (11.0)

### PR DESCRIPTION
**Pull request by bt-admin**
bt_gitbot

### CODE REVIEW

File: base_suspend_security/README.rst [hunk 1]
```diff
@@ -8,6 +8,12 @@ This module was written to  allow you to call code with some `uid` while being s
 
 Usually, you'll be in in the situation to want something like this if you inherit from a module you can't or don't want to change, and call `super()`.
 
+Installation
+============
+
+This version is only valid for being used since Odoo commit
+9af8b31d921dcccb7bcf230271fd3ef4R4329 (merged on 2019-02-04).
```
Comment (hbrunn) [inline base_suspend_security/README.rst:15]: @pedrobaeza this is red on runbot, is there some docker image that needs to be regenerated to have this commit there too?

Comment (pedrobaeza) [reply base_suspend_security/README.rst:15]: I don't think so, as the runbot is built from latest OCB sources. Are you sure is due to this?

Comment (hbrunn) [reply base_suspend_security/README.rst:15]: this is how I understand https://runbot.odoo-community.org/runbot/build/3359255

Comment (pedrobaeza) [reply base_suspend_security/README.rst:15]: That was the previous error before fixing this. Let's try to rebuild the image.

Comment (hbrunn) [reply base_suspend_security/README.rst:15]: ah, wait, sorry for the noise. I didn't rebase on the latest 11 version

Comment (hbrunn) [reply base_suspend_security/README.rst:15]: hmmm, wait again, noise is back: in https://runbot.odoo-community.org/runbot/build/3359265 I added a commit that reverts this change, now the build works on runbot. So my guess is there's indeed some cached image involved

Comment (pedrobaeza) [reply base_suspend_security/README.rst:15]: I have rebuilt main branch and it's correct: https://runbot.odoo-community.org/runbot/repo/github-com-oca-server-backend-253

Comment (hbrunn) [reply base_suspend_security/README.rst:15]: yay, thanks!

**Pull request merged**